docs/07_release_instructions.md

```diff
@@ -0,0 +1,253 @@
+
+## Creating the APK
+
+Now on the remote `master` branch there is the release code which you need to turn into an APK.
+- In the local repository, check out the `master` branch and make sure it is up-to-date with the remote
+    - `git checkout master`
+    - `git pull origin master`
+- Open the local project in Android Studio
+- Run the Gradle `clean` task using Android Studio's interface, in order to cleanup temporary/cache files that may interfere with reproducible builds
+    - Double press Ctrl, type `gradle clean`, press Enter
+- Make sure leftover files from building RC releases are actually removed, in order to avoid confusion
+    - `rm -rf ./app/release`
+- Run the Gradle `assembleRelease` task using Android Studio's interface: it will start the process of building an unsigned APK
+    - Double press Ctrl, type `gradle assembleRelease`, press Enter
+- After a while you should find the APK under `./app/build/outputs/apk/release/app-release-unsigned.apk`
+
+## Having the APK signed by @TheAssassin
+
+Currently @TheAssassin is the only holder of NewPipe's APK signing keys. Therefore you should send the unsigned APK to him and he will send a signed one back. He will also then publish the signed APK in NewPipe's F-Droid repo.
+- Rename `app-release-unsigned.apk` to `NewPipe_vX.X.X.apk`
+- Generate a signature for the APK file
+    - `gpg -b NewPipe_vX.X.X.apk` will generate `NewPipe_vX.X.X.apk.sig`
+    - It will also output 'using "FINGERPRINT" as default secret key for signing'; keep track of the `FINGERPRINT` part
+- Send an email to @TheAssassin and attach both `NewPipe_vX.X.X.apk` and `NewPipe_vX.X.X.apk.sig`
+- If @TheAssassin does not already know it, send him your PGP key `FINGERPRINT` you obtained before
+    - You should send it not using email this time, but another service on which @TheAssassin can be almost sure it is really you (something like 2FA)
+    - For example, either just send it on the IRC group, or create a GitHub gist with the fingerprint and then give that link to @TheAssassin
+- Notify him on IRC that you have sent him an email
+- He will send you back the signed APK
+- Make sure its name is still `NewPipe_vX.X.X.apk` (rename if it's not the case)
+- Install it on your device to see if everything went well (note that installation will work only if your currently installed version of newpipe comes from NewPipe's F-Droid repo or GitHub)
+- Tell @TheAssassin to "push the buttons", i.e. publish the signed APK in NewPipe's F-Droid repo.
+
+## Publishing the release
+
+- Go to the draft changelog [kept on GitHub](https://github.com/TeamNewPipe/NewPipe/releases)
+- Set `vX.X.X` as the tag name
+- Set `vX.X.X` as the release title
+- Set `dev` as the "Target:" branch
+- Attach the signed APK @TheAssassin sent you
+- Publish the release
+- Profit :-D
+
+## Blog post
+
+> I do not know enough about blog post writing and publishing to fill in this section, I'll leave it to @opusforlife2 and @Poolitzer.
```

Comment:
```suggestion
The blog post writers need an up-to-date list of merged PRs numbered in chronological order. This is so that they can keep track of what changes have already been detailed in the draft blog post, and which ones still need to be added. So make sure that there is always at least one up-to-date "master copy" of the draft release notes available for them to review.

The blog post should ideally be published before the GitHub release is made (so that the link to it works!), but in case of some delay, it is fine to let the blog post come later. It is far more important to get the release into users' hands sooner.
```